webshare/share-unicode-strings-nonutf8-manual.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="iso-8859-1">
+    <title>WebShare Manual Test: Share with non-ASCII Unicode strings in a Latin-1-encoded page</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="resources/manual-helper.js"></script>
+  </head>
+  <body>
+    <script>
+        setup({explicit_timeout: true});
+
+        // Exact same test as in share-unicode-strings-manual.html, with same
```

Comment:
To verify the preconditions, can you `assert_equals(document.characterSet, something)`? HTTP headers can ruin it, as could typos in various places.